namespaces/api-namespace.md

```diff
@@ -42,6 +42,11 @@ For example a `category` element may be classified both as `resourceGroup` and
             - resourceGroup - Category is a set of resource.
             - dataStructures - Category is a set of data structures.
             - scenario - Reserved. Category is set of steps.
+- `attributes`
+    - `meta` (object) - Arbitrary metadata
+        - `user` (array[String Pair]) - User-specific metadata. Metadata written in the source.
+        - `adapter` (array[String Pair]) - Serialization-specific metadata. Metadata provided by adapter.
```

Comment:
In other words, `Member Element` are elements have a content for key/value, whereas this is simply an object.